shell/common/animator.h

```diff
@@ -22,6 +22,10 @@ namespace testing {
 class ShellTest;
 }
 
+/// Executor of animations on a |LayerTree|.
+///
+/// This typically happens after the Layout and Paint passes in the Dart code
+/// and before the LayerTree is Rasterized.
```

Comment:
The functions of this component are completely divorced from the inner working of user and framework code in Dart. Also, stuff like `RequestFrame` happens before any of those phases in Dart code. So the final statement is inaccurate.

Something like "In conjunction with the system vsync waiter, allows callers (typically Dart code) to schedule work that ends up generating a layer tree. In cases where callers (Dart Code) end up specifying too much work to the layer tree consumer (the renderer), a backoff mechanism is implemented (with reschduled workloads still lining up with system vsync intervals)" is more accurate.

Reply:
Animator::Render is called on the UI thread after flushLayout and flushPaint as a result of the compositeFrame method.  I took out those details because i think class docstrings should talk about what the class is, not what it does.